source/client-side-encryption/tests/README.rst

```diff
@@ -610,15 +610,15 @@ Using ``client_encrypted`` perform the following operations:
 
    - ``{ "_id": "over_2mib_2", "unencrypted": <the string "a" repeated (2097152) times> }``
 
-   Expect the bulk write to succeed and split after first doc (i.e. two inserts occur). This may be verified using `command monitoring <https://github.com/mongodb/specifications/tree/master/source/command-monitoring/command-monitoring.rst>`_.
```

Comment:
nit: here and below and I would keep the title as "command monitoring" (while updating the link) as I think drivers are likely to use monitoring, not logging, to assert this.